src/test/setup.ts

```
@@ -0,0 +1,81 @@
+
+  return {
+    useCookies: () => [
+      {
+        token: 'mock-token',
+        role: 'ADMIN',
+      },
+      vi.fn(),
+    ],
+    Cookies: class Cookies {
+      constructor() {
+        return mockCookiesInstance;
```

Biome (2.1.2)

[error] The constructor should not return a value.

The constructor is here:

Returning a value from a constructor may confuse users of the class.

(lint/correctness/noConstructorReturn)